people.md

```diff
@@ -7,4 +7,8 @@ We are each one of:
   commitment. When they choose to make commitments they honour them. They can work on projects and
   provide input to the direction and development of dvp, while slowly strengthening their
   relationship with dvp, with the hope of the partners that they eventually will want to become a
-  partner.
\ No newline at end of file
+  partner.
+
+There is a further category of _non-member_ or _freelancer_.
```

Comment:
Suggestion: "There is a further category of people: _non-member_ or similarly _freelancer_."